doc/reference/reference_lua/popen.rst

```diff
@@ -32,6 +32,54 @@ object:
 Either function returns a handle which we will call ``popen_handle`` or ``ph``.
 With the handle one can execute methods.
 
+===============================================================================
+                                    Index
+===============================================================================
+
+Below is a list of all ``popen`` functions.
```

Comment:
Technically module functions and handle methods are listed here, not only functions.

Should not also  **Handle fields** and **Module constants** sections be listed here?

Reply:
Fixed in 61ad4a5